stream.go

```diff
@@ -87,7 +87,8 @@ type ClientStream interface {
 	// Context returns the context for this stream.
 	//
 	// It should not be called until after Header or RecvMsg has returned. Once
-	// called, subsequent client-side retries are disabled.
+	// called, subsequent client-side retries are disabled. It is also not safe
+	// to call CloseSend concurrently with SendMsg.
```

Comment:
This is still on Context().  Move to CloseSend() please.

Reply:
Omg.. Done.